src/smart-components/group/policy/add-policy/remove-policy-modal.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Modal, Button, Bullseye, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/';
+import { fetchGroupPolicies, fetchPolicy, removePolicy } from '../../../../redux/actions/policy-actions';
+
+const RemovePolicyModal = ({
+  history: { goBack, push },
+  removePolicy,
+  fetchPolicy,
+  fetchGroupPolicies,
+  policyId,
+  policy
+}) => {
+  useEffect(() => {
+    if (policyId) {
```

Comment:
What if there is no policy id? This effect should be called also when id is changed.